api_docs/incoming-webhooks-walkthrough.md

```diff
@@ -475,20 +479,24 @@ When you have finished your webhook integration and are ready for it to be
 available in the Zulip product, follow these steps to prepare your pull
 request:
 
+{start_tabs}
```

Comment:
This should also maybe be rewritten as tips / suggestions instead of a set of instructions.

Reply:
How about:

![image](https://github.com/zulip/zulip/assets/2343554/4897c0db-53e1-44a5-8249-12049717b83f)

Reply:
This looks good to me.